classpath/javax/net/ssl/SSLServerSocket.java

```diff
@@ -0,0 +1,33 @@
+package javax.net.ssl;
+
+import java.io.*;
+import java.net.*;
```

Comment:
I agree, in general we like to not use wildcard imports. Here's a brief stackoverflow on a couple of things that could happen if we did. 
http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad